src/utils/worker.ts

```diff
@@ -23,7 +23,7 @@ export const worker = async (): Promise<unknown> => {
     const { repo, enableDependabot, enableSecretScanning, createIssue } =
       repos[index];
     await enableGHAS(repo, client);
-    if (enableDependabot) {
+    if (enableDependabot || process.env.GHES == "true") {
```

Comment:
So we always enable Dependabot if we are working on GHES? Don't we want to skip it as the API is not supported on server?

Reply:
ah this should be `process.env.GHES != "true"` 🤦